src/LayerPlan.cpp

```diff
@@ -646,15 +646,14 @@ void LayerPlan::addWall(ConstPolygonRef wall, int start_idx, const GCodePathConf
         }
     }
 
-    Point p0 = wall[start_idx];
-    addTravel(p0, always_retract);
```

Comment:
I think that this was a much nicer/simpler solution than the business you have below with the `first_line` boolean variable. Why was it removed?

Reply:
Because, before, it would always do the first travel even when the whole wall is not printed because all the line segments have a flow that is too low. You only want to do the initial travel move if there are any segments that are to be printed.